tests.ml

```diff
@@ -571,6 +570,290 @@ let do_test_codemotion = function () ->
   assert_equal (Disasm.disassemble res) (Disasm.disassemble t);
   ()
 
+let do_test_pull_drop () =
+  let open Rewrite in
+  let main = List.assoc "main" in
+  let test input pc x expected =
+    let input = main input in
+    let instrs = match Rewrite.pull_drop input x pc with
+      | Pulled_to ((instrs, _), _) -> instrs
+      | Blocked -> input in
+    (* Printf.printf "%s:\n%s\n"
```

Comment:
Is this printing for debugging? Or should it just be removed (instead of being commented out)?

There's another instance of this below, for the push drop tests.